index.d.ts

```diff
@@ -66,6 +70,13 @@ export class Registry {
 	 */
 	contentType: string;
 
+	/**
+	 * Set the content type of a registry. Used to change between Prometheus and
+	 * OpenMetrics versions.
+	 * @param contentType The type of the registry
+	 */
+	setContentType(contentType: string): void;
```

Comment:
should we type this stricter? I.e. only allow the two known values?

Reply:
I have added a stricter typing to the, let me know what you think. I used string template types from typescript in case the content type will have new versions/encodings in the future.